utils/notebook_utils.py

```diff
@@ -28,6 +28,27 @@
 # In[ ]:
 
 
+def device_widget(default="AUTO", exclude=None):
+    import openvino as ov
+    import ipywidgets as widgets
+
+    core = ov.Core()
+
+    support_devices = core.available_devices+ ["AUTO"]
+    exclude = exclude or []
+    if exclude:
+        for ex_device in exclude:
+            if ex_device in support_devices:
+                support_devices.remove(ex_device)
+
+    device = widgets.Dropdown(
+        options=supported_devices,
```

Comment:
`support_devices `was defined, not `supported_devices`

Reply:
fixed